### PR TITLE
Auth0 Go Live

### PR DESCRIPTION
**Pull request by kutoft**
NOTE: please do not merge until Gabi gives the approval for go live.

### CODE REVIEW

File: packages/backend/src/lib/date-utils.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import dayjs from 'dayjs'
 import dayJsutcPlugin from 'dayjs/plugin/utc'
+dayjs.extend(dayJsutcPlugin)
```
Comment (kutoft) [inline packages/backend/src/lib/date-utils.ts:3]: this doesn't make any sense, this code is already in "mvp" branch but this PR is acting like its not.

File: packages/backend/src/known-chains.ts [hunk 2]
```diff
@@ -148,5 +148,15 @@ export const KNOWN_CHAINS = {
     ticker: 'MOVR',
     id: '0051',
     limit: 0,
+  },
+  OPTIMISM_MAINNET: {
```
Comment (kutoft) [inline packages/backend/src/known-chains.ts:152]: weird, this code is already in "mvp" branch but this PR is acting like its not.

File: packages/frontend/src/lib/chain-utils.ts [hunk 3]
```diff
@@ -221,11 +226,29 @@ export const CHAIN_ID_PREFIXES = new Map<string, ChainMetadata>([
...
-    { prefix: 'moonriver-mainnet', name: 'MOONRIVER Mainnet', abbrv: 'MOONRIVER', evm: true },
+    {
+      prefix: 'moonriver-mainnet',
+      name: 'MOONRIVER Mainnet',
+      abbrv: 'MOONRIVER',
+      evm: true,
+    },
+  ],
+  [
+    '0053',
```
Comment (kutoft) [inline packages/frontend/src/lib/chain-utils.ts:246]: weird, this code is already in "mvp" branch but this PR is acting like its not.

**Review by commoddity — COMMENTED** [review]
LGTM - Will add the auth0 regex change in a separate branch.

File: packages/backend/src/App.ts [hunk 4]
```diff
@@ -12,28 +11,28 @@ import { connect } from './db'
 const PORT = process.env.PORT || 4200
 const ALLOWED_DOMAINS = env('ALLOWED_DOMAINS') as unknown as string[]
 
+if (!env('PROD')) {
+  ALLOWED_DOMAINS.push('http://localhost:3001')
```
[the remark this answers is not in the document]
Comment (commoddity) [reply packages/backend/src/App.ts:15]: Yes I agree with this one.

File: packages/backend/src/db.ts [hunk 5]
```diff
@@ -2,27 +2,30 @@ import mongoose from 'mongoose'
 import env from './environment'
 
 const DEV_DB_URL =
-  'mongodb://mongo-client:mongo-password@localhost:27017/gateway-testnet?authSource=admin&readPreference=primary&appname=MongoDB%20Compass&directConnection=true&ssl=false'
+  'mongodb://mongouser:mongopassword@localhost:27017/gateway?authSource=admin'
```
[the remark this answers is not in the document]
Comment (commoddity) [reply packages/backend/src/db.ts:5]: Also agreed.

**Review by jhodak — APPROVED** [review]

**Review by carlosvq — APPROVED** [review]

**Pull request merged**